Gemfile.lock

```diff
@@ -445,7 +445,7 @@ GEM
     ruby-progressbar (1.11.0)
     ruby-rc4 (0.1.5)
     ruby2_keywords (0.0.5)
-    ruby_smb (3.1.1)
+    ruby_smb (3.1.2)
```

Comment:
cc @zeroSteiner

Is this bump blocked on https://github.com/rapid7/metasploit-framework/pull/16481 - or can this go out independently 👀

Reply:
That should be fine to go out. There are no breaking changes that I'm aware of.

It includes:
* rapid7/ruby_smb#213
* rapid7/ruby_smb#214
* rapid7/ruby_smb#215
* rapid7/ruby_smb#216
* rapid7/ruby_smb#217